huaweicloud/services/acceptance/cc/data_source_huaweicloud_cc_central_network_policies_test.go

```diff
@@ -0,0 +1,220 @@
+package cc
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccDataSourceCcCentralNetworkPolicies_basic(t *testing.T) {
+	dataSource := "data.huaweicloud_cc_central_network_policies.test"
+	rName := acceptance.RandomAccResourceName()
+	dc := acceptance.InitDataSourceCheck(dataSource)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
+			acceptance.TestAccPreCheckCCProjectID(t)
+			acceptance.TestAccPreCheckCCRegionName(t)
+		},
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testDataSourceDataSourceCcCentralNetworkPolicies_basic(rName),
+				Check: resource.ComposeTestCheckFunc(
+					dc.CheckResourceExists(),
+					resource.TestCheckResourceAttrSet(dataSource, "central_network_policies.#"),
+					resource.TestCheckResourceAttrSet(dataSource, "central_network_policies.0.id"),
+					resource.TestCheckResourceAttrSet(dataSource, "central_network_policies.0.status"),
+					resource.TestCheckResourceAttrSet(dataSource, "central_network_policies.0.central_network_id"),
+					resource.TestCheckResourceAttrSet(dataSource, "central_network_policies.0.created_at"),
+
+					resource.TestCheckOutput("id_filter_is_useful", "true"),
+					resource.TestCheckOutput("status_filter_is_useful", "true"),
+					resource.TestCheckOutput("is_applied_filter_is_useful", "true"),
+				),
+			},
+		},
+	})
+}
+
+func testDataSourceDataSourceCcCentralNetworkPolicies_basic(name string) string {
+	return fmt.Sprintf(`
+%s
+
+data "huaweicloud_cc_central_network_policies" "test" {
+  depends_on = [
+    huaweicloud_cc_central_network_policy.test2,
+    huaweicloud_cc_central_network_policy_apply.test,
+  ]
+ 
+  central_network_id = huaweicloud_cc_central_network.test.id
+}
+	  
+locals {
+  central_network_policies = data.huaweicloud_cc_central_network_policies.test.central_network_policies
+  id                       = local.central_network_policies[0].id
+  status                   = local.central_network_policies[0].status
+}
+
+data "huaweicloud_cc_central_network_policies" "filter_by_id" {
+  central_network_id = huaweicloud_cc_central_network.test.id
+  policy_id          = local.id
+}
+
+data "huaweicloud_cc_central_network_policies" "filter_by_status" { 
+  central_network_id = huaweicloud_cc_central_network.test.id
+  status             = local.status
+}
+
+data "huaweicloud_cc_central_network_policies" "filter_by_applied" {
+  depends_on = [
+    huaweicloud_cc_central_network_policy.test2,
+    huaweicloud_cc_central_network_policy_apply.test,
+  ]
+
+  central_network_id = huaweicloud_cc_central_network.test.id
+  is_applied         = "true"
+}
```

Comment:
```suggestion
  central_network_id = huaweicloud_cc_central_network.test.id
    is_applied = "true"
  }
```
and fix other places